src/DoctrineMigrations/Version20210910203121.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210910203121 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE file_item DROP FOREIGN KEY FK_96E74D70FC6B21F1');
+        $this->addSql('ALTER TABLE issue DROP FOREIGN KEY FK_12AD233EC38008F2');
+        $this->addSql('ALTER TABLE log_entry DROP FOREIGN KEY FK_B5F762DA76ED395');
+        $this->addSql('ALTER TABLE report DROP FOREIGN KEY FK_C42F7784F675F31B');
+        $this->addSql('CREATE TABLE users (id INT AUTO_INCREMENT NOT NULL, institution_id INT NOT NULL, username VARCHAR(180) NOT NULL, name VARCHAR(255) DEFAULT NULL, roles JSON DEFAULT NULL, lms_user_id VARCHAR(128) NOT NULL, api_key VARCHAR(2048) DEFAULT NULL, refresh_token VARCHAR(512) DEFAULT NULL, created DATETIME NOT NULL, last_login DATETIME NOT NULL, UNIQUE INDEX UNIQ_1483A5E9F85E0677 (username), INDEX IDX_1483A5E910405986 (institution_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+        $this->addSql('ALTER TABLE users ADD CONSTRAINT FK_1483A5E910405986 FOREIGN KEY (institution_id) REFERENCES institution (id)');
+        $this->addSql('DROP TABLE user');
```

Comment:
Can we alter the existing table name instead of creating a new one and dropping the old one?  I don't want existing installs to lose all of their user data.

Reply:
Replaced all of that with a simple rename command